pkgs/by-name/kn/knxd/package.nix

```diff
@@ -60,11 +60,11 @@ stdenv.mkDerivation (finalAttrs: {
     "systemdsysusersdir=$(out)/lib/sysusers.d"
   ];
 
-  meta = with lib; {
+  meta = {
     description = "Advanced router/gateway for KNX";
     homepage = "https://github.com/knxd/knxd";
-    license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ sikmir ];
-    platforms = platforms.unix;
+    license = lib.licenses.gpl2Plus;
+    maintainers = with lib.maintainers; [ sikmir ];
```

Comment:
Please do this directy, without the `with`.

Reply:
Any reason why?
```sh
$ ag -c "maintainers = with lib.maintainers;" | wc -l
    6100
$ ag -c "maintainers = \[ lib.maintainers." | wc -l
    1327
```

Reply:
There is only one maintainer.

`with` is normally used when there is like three or more.

`with` is also a little overused and make static analysis more difficult.

Reply:
Is there any consensus on it? Maybe discussed somewhere? Because many reviewers asks to use `with lib.maintainers` even if only one maintainer.

Reply:
It seems like the nitpick issue is back lol

Reply:
https://nixos.org/manual/nixpkgs/stable/#chap-meta also recommends `with lib.maintainers`.